events/tests/snapshots/snap_test_api.py

```diff
@@ -524,6 +524,46 @@
     }
 }
 
+snapshots["test_events_and_event_groups_query_upcoming_filter[False-False] 1"] = {
+    "data": {
+        "eventsAndEventGroups": {
+            "edges": [{"node": {"__typename": "EventNode", "name": "In the future"}}]
+        }
+    }
+}
+
+snapshots["test_events_and_event_groups_query_upcoming_filter[False-True] 1"] = {
+    "data": {
+        "eventsAndEventGroups": {
+            "edges": [
+                {"node": {"__typename": "EventGroupNode", "name": "In the future"}}
+            ]
+        }
+    }
+}
+
+snapshots["test_events_and_event_groups_query_upcoming_filter[True-False] 1"] = {
+    "data": {
+        "eventsAndEventGroups": {
+            "edges": [
+                {"node": {"__typename": "EventNode", "name": "Within leeway"}},
+                {"node": {"__typename": "EventNode", "name": "In the future"}},
```

Comment:
Nice!

Reply:
Yup, thought it's easier to check the snapshots for correct results.